### ISSUE TITLE
build-windows-msvc is broken

### ISSUE THREAD

**Issue by rexim** [tsoding/ded#29]
introduced in #25

**Comment by Tschitschibor** [comment]
"ded\dependencies\SDL2\include\SDL2\SDL2\begin_code.h"

Is it correct that there are two nested SDL2 folders?

**Comment by kolumb** [comment]
@Tschitschibor No, it looks like that you launched `setup_dependencies.bat` second time. Delete `dependencies` folder first and only then launch `setup_dependencies.bat`.
Fixing it would complicate bat file even more. It's already planned to be changed so there's no need of setup step.

**Comment by IRooc** [comment]
The latest code is building in both CI and locally in this PR https://github.com/tsoding/ded/pull/42
